guides/customize_tinymce.md

```diff
@@ -73,6 +73,41 @@ That's it. Now the editor can format a block of text as headline 2. Nice!
 Custom buttons always appear in all richtext editors in all elements in Alchemy.
 :::
 
+### Adding editor plugins
+It's possible to configure editor with it's plugins.
+All you have to do is copy (for example) the tinymce list plugin folder to 
+~~~
+vendor/assets/javascripts/tinymce/plugins
+~~~
+
+and add the following content to config/initializers/tinymce.rb
+
+~~~
+Alchemy::Tinymce.init = {
+    plugins: Alchemy::Tinymce.plugins + ['lists'],
```

Comment:
The is indentation is wrong in this example. It should be two spaces instead of four. Also use another plugin as example, because the lists plugin is already shipped with Alchemy